### DISCUSSION TITLE
Cannot search by "ghost" author

### DISCUSSION THREAD

**Discussion by rlwimi** [community/community#14009] [Pull Requests]
The ability to search for `is:pr author:ghost` or some other filter criteria like author-is-deleted and get a list of PRs authored by any deleted account would be wonderful. I'm interested in similar functionality based on filtering on `ghost`, but to a lesser extent.

I occasionally need to find a PR authored by someone who deleted their account. PR filtering on `author` doesn't work for the username of an account that was deleted. I can understand this, but, in a long historical list of PRs, when I don't remember the time frame, it takes an inordinate amount of time to manually dig around to find the PR.

**Comment by ghost** [comment]
I vote for this one. I found out today after deleting a dummy account I created for a private repo, that now even though the commits are associated to my account (since the email wasn't the `noreply` one and I just added it to the regular account) on the PR view, I cannot search for the commits nor the PRs since they are associated to the ghost user now. Really annoying!

**Reply by wdhdev** [reply]
*Is now a ghost user*

**Reply by rogerluan** [reply]
The fact that this user is now a ghost makes this sooooooo much better 😂😂

**Comment by a7medelbatal** [comment]
i don't understand you..

**Comment by kglickman** [comment]
Is a ghost user someone who no longer has access to the repo?  I frequently find myself wanting to filter on issues and prs that were authored by someone who is no longer on my team.

**Reply by UnBeatWaterGH** [reply]
They are users who have deleted their GitHub account.
